src/seedu/addressbook/commands/Command.java

```diff
@@ -63,6 +63,10 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    public boolean isMutating() {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.